src/analysis/special_functions/pow.lean

```diff
@@ -1361,143 +1364,84 @@ begin
   { simp [coe_rpow_of_nonneg _ (nat.cast_nonneg n)] }
 end
 
-@[norm_cast] lemma coe_mul_rpow (x y : ℝ≥0) (z : ℝ) :
-  ((x : ℝ≥0∞) * y) ^ z = x^z * y^z :=
```

Comment:
Is this lemma worth keeping in some form?

Reply:
Yes, see [line 1389](https://github.com/leanprover-community/mathlib/pull/8308/files#diff-454e0a05eeada91b4b5af62958580c173fc62e7a52d968b60c3aa6070253b4c1R1389).